home-data/src/main/java/com/softserveinc/ita/homeproject/model/Permission.java

```diff
@@ -0,0 +1,34 @@
+package com.softserveinc.ita.homeproject.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+public class Permission {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @ManyToMany(mappedBy = "permissions")
+    @EqualsAndHashCode.Exclude
```

Comment:
We don't have anything to exclude here. Also if we have a primary key it is the only candidate for equals and hashcode.